app/views/layouts/_nav_bar.html.erb

```diff
@@ -1,22 +1,21 @@
 <%
-  t = @teams.nil? ? nil : @teams.map(&:updated_at).join;
+  t = @teams.nil? ? nil : @teams.map(&:updated_at).join
+  fab_controller = params[:controller] == "users"
+  list_action = params[:action] == "index"
   nav_horoscope = [
-       params[:controller] == "users",
-       params[:action] == "index",
-       current_user.try(:admin?),
-       user_signed_in?,
-       t
-     ];
+    fab_controller, list_action, current_user.try(:admin?), user_signed_in?, t
+  ]
+  on_fabs_list = (fab_controller and list_action)
 %>
 
 <% cache(Digest::MD5.hexdigest(nav_horoscope.join)) do %>
 
-
-  <nav><img src="<%= asset_path('EFF-logo-plain.svg') %>" type="image/svg+xml">
+  <nav>
+    <img src="<%= asset_path('EFF-logo-plain.svg') %>" type="image/svg+xml">
 
     <ul>
 
-      <% if params[:controller] == "users" and params[:action] == "index" %>
+      <% if on_fabs_list %>
 
         <% cache(Digest::MD5.hexdigest(['leet-filter-select', t].join)) do %>
```

Comment:
I was going to suggest you take this opportunity to be a hero and remove all occurrences of the word "leet" from this file, but it turns out it's used all over the project, so I suppose these cache keys are okay...

Reply:
Let's get them later though :)